cachi2/core/package_managers/yarn/main.py

```diff
@@ -174,9 +175,12 @@ def _get_plugin_allowlist(yarn_rc: YarnRc) -> list[Plugin]:
     This list should only have official plugins that add new protocols and that also do not
     implement the 'fetchPackageInfo' hook, since it would allow arbitrary code execution.
 
+    Note that starting from v4, the official plugins are enabled by default and can't be disabled.
```

Comment:
Do we need to be concerned by any of the official plugins (disclaimer: I haven't checked the full set of plugins)?

Reply:
AFAICT, no, we don't. I wrote a small section about the official plugins and their functions in the ADR to try to TL;DR  the reason why I believe this.